src/content/guide/how-to-build-a-product/security.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Security
+columns: two
+template: guide.hbs
+order: 5
+---
+
+# Security
+
+When it comes to IoT devices, there is no shortage of security concerns. This
+part of the guide will be a discussion of security-related topics that arise
+with a product at scale.
+
+## Treating Devices as Products
+
+When you created a product, you created a new type of device in the Particle
+cloud. So far, development kits that you have been using for prototyping have been considered as a "Photon," "P1", "Electron," or "Core" by the
+Particle cloud. However, when you added devices to your product, you are signaling to the cloud
+that you would like them to be treated as part of your product instead of
+as development kits.
+
+There are certain security implications that arise around devices behaving as products. Ideally, the following is always true:
+- Only the _devices_ that be given product privileges will recieve them
+- Only the _people_ who should have the ability to interact with the
+  product device will be allowed to do so
```

Comment:
Only the people authorized to interact with the product device will be able to do so